app/adapters/application.js

```diff
@@ -0,0 +1,6 @@
+import JSONAPIAdapter from '@ember-data/adapter/json-api';
+
+export default class ApplicationAdapter extends JSONAPIAdapter {
+  host = 'http://localhost:3000';
+  namespace = 'api/v1';
```

Comment:
All the rest of the JSONAPIAdapter defaults do the things that we want, at least right now. We can create a UserAdapter or a JobAdapter down the road that do custom things before sending or after receiving the data.

But here we tell it that all of our adapters are going to go to this host and this namespace for API requests. In the Rails app, you'll see the controller endpoints are in the `api/v1` folder, which is a common pattern for APIs. One day we might want a version 2 of the API for some reason, especially if it was a public API for developers to build off of.

Reply:
When we actually deploy, we'll need the `host` to be an environment variable, so it's http://localhost:3000 locally but https://api.bristle.work, or something like that, on production.